.gitignore

```diff
@@ -62,9 +61,11 @@ examples/jbnmr_examples/s10_covariance_processing/test.ft
 examples/jbnmr_examples/s7-s9_s3e_processing/fid
 examples/jbnmr_examples/s7-s9_s3e_processing/procpar
 
-
 #Eclipse PyDev
 .project
 .pydevproject
 *.prefs
+
+#Pycharm dev
 .idea
+*.egg-info
```

Comment:
This is OK for now I guess, but given the increasing number of editors that use hidden folders, I think I would recommend putting a `.gitignore` file with a `*` inside the `.idea` or `.vscode` or similar folder so that these folders are automatically ignored by git. Or, one can use `$GIT_DIR/.git/info/exclude` to exclude these folders

Reply:
I can remove this and declare these exclusion patterns as global.